desc/integrals/bounce_integral.py

```diff
@@ -422,121 +622,154 @@ def compute_theta(
         params=None,
         profiles=None,
         tol=1e-7,
+        maxiter=30,
+        **kwargs,
+    ):
+        """Method has been deprecated in favor of Bounce2D.angle."""
+        warnings.warn("Please use Bounce2D.angle instead.", DeprecationWarning)
+        return Bounce2D.angle(
+            eq, X, Y, rho, iota, params, profiles, tol, maxiter, **kwargs
+        )
+
+    @staticmethod
+    def angle(
+        eq,
+        X=16,
+        Y=32,
+        rho=jnp.array([1.0]),
+        iota=None,
+        params=None,
+        profiles=None,
+        tol=1e-7,
+        maxiter=30,
         **kwargs,
     ):
-        """Return DESC coordinates θ of (α,ζ) Fourier Chebyshev basis nodes.
+        """Return the angle for mapping boundary coordinates to field line coordinates.
```

Comment:
what does "boundary coordinates" refer to here?

Reply:
Boundary coordinates are DESC theta and zeta always. I figured boundary is easier to swallow than understanding our symbols.